src/main.ts

```diff
@@ -6,8 +7,10 @@ import { REDACT_STRINGS } from './constants';
 import { ConsoleLoggerWithRedact } from './logging/console-logger-with-redact';
 
 const main = async () => {
-  const logger = new ConsoleLoggerWithRedact(REDACT_STRINGS);
-  const nestApp: NestApplication = await NestFactory.create(MainModule, { logger, bufferLogs: false });
+  // If REDACT_LOGS is true use ConsoleLoggerWithRedact. ConsoleLoggerWithRedact is used just if `NestFactory.create` is fails completely.
```

Comment:
I don't think we need the last `is` in this sentence :)

Reply:
`Is` definitely is not required